src/sql/workbench/browser/modelComponents/declarativeTable.component.ts

```diff
@@ -177,11 +184,11 @@ export default class DeclarativeTableComponent extends ContainerBase<any> implem
 		let column: azdata.DeclarativeTableColumn = this.columns[colIdx];
 		let cellData = this.data[rowIdx][colIdx];
 		if (cellData && column.categoryValues) {
-			let category = find(column.categoryValues, v => v.name === cellData);
+			let category = find(column.categoryValues, v => v.name === cellData.value);
 			if (category) {
 				return category.displayName;
 			} else if (this.isEditableSelectBox(colIdx)) {
-				return cellData;
+				return String(cellData.value);
```

Comment:
is there a reason we return a String object instead of a string literal?

Reply:
String() creates a string literal (thanks Javascript for that), new String() would create an object.